Gems/ScriptCanvasTesting/Code/Source/Framework/ScriptCanvasTestUtilities.cpp

```diff
@@ -83,22 +83,22 @@ namespace ScriptCanvasTests
 
         if (!reporter.IsGraphLoaded())
         {
-            ADD_FAILURE() << "Graph was not successfully loaded.";
+            ADD_FAILURE() << "Graph was not successfully loaded.\n" << reporter.GetFilePath().c_str();
```

Comment:
We have a `PrintTo` function overload for GTest for O3DE string classes that allows pretty printing using AZStd types (https://github.com/o3de/o3de/blob/development/Code/Framework/AzTest/AzTest/Printers.h#L14-L31)